blog/2024-03-07-pi4micronaut-announcement.md

```diff
@@ -0,0 +1,25 @@
+---
+title: What's cooking at Open Source with SLU
+description: We are coming up with a special recipe on the occasion of Pi Day i.e,. on March 14.
+slug: pi4micronaut
+authors: [ruthvik]
+tags: [announcement, news]
+image: ./pi4micronaut-20240307/baking@oss-slu.png
+hide_table_of_contents: false
+---
+
+We are coming up with a special recipe on the occasion of Pi Day i.e,. on March 14.
```

Comment:
I'd suggest simplifying this sentence for clarity and impact, something like: "We are coming up with a special recipe on the occasion of Pi Day, March 14."